sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1067,11 +1067,8 @@ synchronized void getLatestReleaseDetails(final String distributionGroupId, Stri
         }
         String releaseHash = computeReleaseHash(mPackageInfo);
         String url = mApiUrl;
-        if (updateToken == null) {
-
-            /* TODO Remove this group and use new endpoint. The hardcoded is temporary to keep testing on sasquatch int. */
-            String publicDistributionGroupId = distributionGroupId == null ? "3d054d79-8b26-426c-9a49-fed752c777d2" : distributionGroupId;
-            url += String.format(GET_LATEST_PUBLIC_RELEASE_PATH_FORMAT, mAppSecret, publicDistributionGroupId, releaseHash, getReportingParametersForUpdatedRelease(true, ""));
+        if (getUpdateTrack() == UpdateTrack.PUBLIC) {
```

Comment:
Please keep the in memory check with token (even if we have a token we don't pass it to this function if public mode, this is done in another function), otherwise this will be subject to value changes during current workflow and various other race conditions.

Reply:
```suggestion
        if (updateToken == null) {
```